### ISSUE TITLE
[Components] Spinner

### ISSUE THREAD

**Issue by andresz1** [adevinta/spark#516]
**Story definition**

As a spark user, I want to have a Spinner component, so I can provide feedback that something async is happening

**Acceptance criteria**

- Should have an accessibility label visually hidden
- Should be able to change the size
- Should be able to change the color

Example:

```jsx
<Spinner intent="primary" size="md" label="Loading" />
```

**Comment by turolopezsanabria** [comment]
✅ OK design review
